crates/symbolicator-js/src/interface.rs

```diff
@@ -77,6 +77,31 @@ pub struct JsModuleError {
     pub kind: JsModuleErrorKind,
 }
 
+/// A JS module (representing a minified file and sourcemap).
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq, PartialOrd, Ord)]
+pub struct JsModule {
+    /// The type of the module.
+    ///
+    /// This field carries no useful information because `JsModuleType`
+    /// only has one variant. It is used to distinguish `JsModule`s from
+    /// JVM and native modules during deserialization.
+    pub r#type: JsModuleType,
+    /// The module's minified code file.
+    pub code_file: String,
+    /// The module's debug ID.
+    pub debug_id: String,
```

Comment:
Wouldn’t it make sense to make this an `Option`?

Reply:
You mean just to be sure? Sentry already only sends modules that have a debug id.

Reply:
oh right, because otherwise there is no point in defining a `module` and we will just use the `code_file` as it appears in the stacktrace?